lib/ui/settings_page.dart

```diff
@@ -118,10 +111,10 @@ class SettingsPage extends StatelessWidget {
 
     return SingleChildScrollView(
       child: Padding(
-        padding: const EdgeInsets.symmetric(vertical: 12.0, horizontal: 20),
+        padding: const EdgeInsets.fromLTRB(16, 16, 16, 24),
         child: Center(
           child: ConstrainedBox(
-            constraints: const BoxConstraints(maxWidth: 350),
+            constraints: const BoxConstraints(maxWidth: 428),
```

Comment:
Can we please test this on Simulator for iPad and SE, if not already done.

Reply:
Did test it on an iPad, tested on SE now and looks fine on both of them.